app/jobs/update_trainee_to_dttp_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class UpdateTraineeToDttpJob < ApplicationJob
+  queue_as :dttp
+
+  discard_on Dttp::ContactUpdate::Error
```

Comment:
Still not sure about suppressing this..it makes sense if we have a way to notify the user/admin but we don't at the moment so wondering whether we let the error blow up the job. Only downside is that it'll start to pile up if it happens again in the next run.

Ideally, would be good to send an email with the error so the data for the record could be fixed and it'll update properly in the next run.

Reply:
I don't think we should notify the user as they should be blissfully unaware of DTTP. We should definitely let it out to Sentry. Not sure whether discarding it is the right thing as we need to check where that ends up in Sidekiq. We'll have to sort out any issues at our end so we'll need to be able to see them.

Reply:
done, i've let the error bubble so we'll know about it in Sentry https://github.com/DFE-Digital/register-trainee-teachers/pull/459/commits/7ff7dd61fba7e1c454ed41eb507755da39a0b506